hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/LocalExecutorStatsImpl.java

```diff
@@ -43,17 +49,17 @@
     private long creationTime;
 
     // These fields are only accessed through the updaters
-    @Probe
+    @Probe(name = EXECUTOR_METRIC_PENDING)
     private volatile long pending;
-    @Probe
+    @Probe(name = EXECUTOR_METRIC_STARTED)
     private volatile long started;
-    @Probe
+    @Probe(name = EXECUTOR_METRIC_COMPLETED)
     private volatile long completed;
-    @Probe
+    @Probe(name = EXECUTOR_METRIC_CANCELLED)
     private volatile long cancelled;
-    @Probe
+    @Probe(name = EXECUTOR_METRIC_TOTAL_START_LATENCY)
```

Comment:
Do we need `MS` unit here and in the next probe?

Reply:
Done